webapp/src/shared/components/Game/Game.tsx

```diff
@@ -1,72 +1,73 @@
 import * as React from 'react';
 
+import socket from '../../socket';
 import { CANVAS_WIDTH, CANVAS_HEIGHT } from './constants';
-import { GameBall, GamePaddle } from './types';
-import { initialBallState, initialPaddleState, isLose, isWin } from './physics';
-import useGameControls from './useGameControls';
-import useGameAnimation from './useGameAnimation';
+import {
+  useGameControls,
+  useGameAnimation,
+  useClientGameEngine,
+} from './hooks';
+import { GameStateContextProvider } from './context';
 
 import './Game.css';
 
-export default function Game() {
+const GAME_SERVER_MESSAGE = 'gameServerMessage';
+
+const Game = () => {
```

Comment:
This component is an exact copy of the `GameTraining` below one. It only have scaffolding ws connection to start working on it